Makefile

```diff
@@ -1,9 +1,9 @@
 all: build test
 
-CGO_ENABLED=0
+export CGO_ENABLED=0
 
 build:
-	go build -trimpath -o bin/schemadiff ./cmd/schemadiff/main.go
+	go build -trimpath -ldflags="-s -w" -o bin/schemadiff ./cmd/schemadiff/main.go
```

Comment:
This reduces the size of the resulting binary as well and we don't really lose much with it.